### ISSUE TITLE
Batch processing

### ISSUE THREAD

**Issue by jherzberg** [argman/EAST#58]
Is it possible to batch process images?

**Comment by argman** [comment]
do you mean when generating training images? I think its possible , you can first generate tf-records.

**Comment by jherzberg** [comment]
No, in eval.py. Can I stack two images and process both. Seems like in the original code, the input tensor had shape (none, none, none, 3) which would suggest this is possible.

**Comment by argman** [comment]
its possible, if your test images all have same shape

**Comment by jherzberg** [comment]
Great, thanks!

**Comment by zxytim** [comment]
To be specific, the forward pass of network can be batched, but not the NMS
cannot be batched. You can use multi-threading to speedup NMS.

**Comment by ansul26** [comment]
can u help in which exact part we can use multi-threading
